site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Configure Identity Providers (IDPs) on a FederationDomain
+description: Learn how to use one or more identity providers, and identity transformations and policies, on a FederationDomain.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: IDPs on FederationDomains
+    weight: 20
+    parent: howto-configure-supervisor
+---
+
+This guide explains how to associate one or more external identity providers (IDPs) with a FederationDomain.
```

Comment:
Elsewhere we link [FederationDomain](https://github.com/vmware-tanzu/pinniped/blob/main/generated/1.27/README.adoc#federationdomain) to this doc.  I think we should do that here as well.  Long term, I think we should add a `Custom Resources` menu item and detail out what each one is.  `FederationDomain` is an important concept, but as I read our docs I get the sense that they are written assuming people understand the concept.

Reply:
Updated.